models/truckModel.js

```diff
@@ -2,14 +2,19 @@
 // when exporting the mongoose model you will create using this schema, 
 // name your model 'Truck' (see the example in the slide deck for 'Book')
 // this is to match the name of the collection the data was imported into
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+var db = mongoose.connect('mongodb://localhost/foodTruckApi');
```

Comment:
My bad! I didn't realize I made that typo; I thought it had come from the repo.  I had even looked at and thought it was strange that API wasn't capitalized...